src/components/ProcessingCard/LinkButton.tsx

```diff
@@ -1,40 +1,31 @@
-import { LinkText } from 'components/ui/Text'
+import { scrollToHashElement } from 'hooks/useScrollToAnchor'
+import { useLocation } from 'wouter'
 import Button from 'components/ui/Button'
 
 interface LinkButtonProps {
   url: string
-  internal?: boolean
   approved?: boolean
-  onClick?: () => void
 }
 
-export default function ({
-  url,
-  internal,
-  approved,
-  onClick,
-}: LinkButtonProps) {
+export default function ({ url, approved }: LinkButtonProps) {
+  const [, setLocation] = useLocation()
+
   const linkTitle = approved
     ? 'View cast on Farcaster'
     : 'View cast on Sealcaster'
 
   return (
-    <LinkText
-      url={url}
-      title={linkTitle}
-      internal={internal}
-      gradientFrom="from-secondary"
-      gradientTo="to-accent"
+    <Button
+      small
+      withArrow
+      gradientFont
+      type="tertiary"
+      onClick={() => {
+        setLocation(url)
```

Comment:
The problem with `Link` wrapping is that location changes after `onClick`, so user need to click twice on the button